Grbl_Esp32/src/SDCard.h

```diff
@@ -40,6 +41,14 @@ class SDCard : public Configuration::Configurable {
         BusyParsing   = 4,
     };
 
+    enum class ReadResult_t : uint8_t {    // return value from readFileLine()
+        OK                  = 0,
+        EndOfFile           = 1,
+        FSError             = 2,
```

Comment:
Since the code that checks the return value does not distinguish between the different error values - only between OK, EOF, and everything else, it would be better to collapse 2,3,and 4 into a single Error return.  The code that detects the various errors already reports why the error happened.